system/hardware/hw.py

```diff
@@ -0,0 +1,28 @@
+import os
+from selfdrive.hardware import PC
+from pathlib import Path
```

Comment:
imports should go stdlib + third party, then our imports

Reply:
done